### PR TITLE
Upgrade all packages

### PR DESCRIPTION
**Pull request by langpavel**

### CODE REVIEW

**Comment by langpavel** [comment]
Babel 7 configuration should be checked

File: package.json [hunk 1]
```diff
@@ -45,56 +45,58 @@
...
   "dependencies": {
-    "codemirror": "^5.26.0",
+    "codemirror": "^5.40.2",
     "codemirror-graphql": "^0.7.1",
-    "markdown-it": "^8.4.0"
+    "markdown-it": "^8.4.2"
   },
   "peerDependencies": {
-    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0",
+    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^0.14.0",
```
Comment (philiptzou) [inline package.json]: ^0.14.0 or ^14.0.0?

Comment (langpavel) [reply package.json]: Uh, you are right..

**Comment by acao** [comment]
@langpavel @philiptzou I'd like to revive this PR in anticipation of the upcoming graphiql working group meeting:

https://github.com/graphql/graphql-wg/issues/136

maybe also bump flow-bin again to 0.98 to catch up with `graphql-js`

**Comment by philiptzou** [comment]
@acao Sounds good to me.

**Comment by acao** [comment]
Here's the diff, merged in the upstream and re-upgraded things

https://github.com/the-gear/graphiql/compare/upgrade-packages...acao:upgrade-packages

**Comment by langpavel** [comment]
@acao Merged. Thaks!

**Comment by acao** [comment]
@langpavel @philiptzou update, so we have been breaking up this PR and merging it as smaller chunks, for purposes of a clean revision history

**Comment by Neitsch** [comment]
Going to close this, given @acao 's comment. Big PRs like this are hard to review. If possible, please break them up into smaller ones :)

**Pull request closed without merging**